src/resources/Indexes/Indexes.ts

```diff
@@ -0,0 +1,58 @@
+import API from '../../APICore';
+import {PageModel} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {CreateCoveoIndexModel, IndexAttributes, IndexBackupsItems, IndexStatisticsModel} from './IndexesInterface';
+
+export default class Indexes extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/indexes`;
+    static indexBackupUrl = `/rest/organizations/${API.orgPlaceholder}/indexbackups/page`;
+
+    list() {
+        return this.api.get<IndexAttributes[]>(Indexes.baseUrl);
+    }
+
+    // Same for Copy Index but making sure to pass copyFromId and machineSpec
+    create(indexModal?: CreateCoveoIndexModel) {
+        return this.api.post<{id: string}>(Indexes.baseUrl, indexModal);
+    }
+
+    delete(indexId: string) {
+        return this.api.delete(`${Indexes.baseUrl}/${indexId}`);
+    }
+
+    get(indexId: string) {
+        return this.api.get<IndexAttributes>(`${Indexes.baseUrl}/${indexId}`);
+    }
+
+    backup(indexId: string) {
+        return this.api.post<{id: string}>(`${Indexes.baseUrl}/${indexId}/backup`, {});
+    }
+
+    getBackups() {
+        return this.api.get<PageModel<IndexBackupsItems>>(`${Indexes.indexBackupUrl}`);
```

Comment:
This call has some query parameters
![image](https://user-images.githubusercontent.com/35579930/67498721-f253be80-f64d-11e9-9ca3-4e120f9ed5c6.png)
You can have a look at `platform-client/src/resources/Groups/Groups.ts` (`update` method) for how to handle those.